### DISCUSSION TITLE
how to change code block theme?

### DISCUSSION THREAD

**Discussion by ghost** [web-infra-dev/rspress#1416] [Q&A] [answered]
The current code block theme doesn't satisfy my needs. Is there a way to change the theme to GitHub Dark or One Dark Pro?

**Comment by Timeless0911** [comment, marked as answer]
use [Custom Styles](https://rspress.dev/guide/basic/custom-page#custom-styles) to override styles.

- https://github.com/web-infra-dev/rspress/blob/a83fe41ce9166231f09791298c90fc63599529c7/packages/theme-default/src/styles/base.css#L69-L102

- https://github.com/web-infra-dev/rspress/blob/main/packages/theme-default/src/styles/code.css
